src/anaconda/install.sh

```diff
@@ -8,10 +8,10 @@
 # Maintainer: The VS Code and Codespaces Teams
 
 
-VERSION=${VERSION:-"latest"}
-USERNAME=${USERNAME:-"automatic"}
-UPDATE_RC=${UPDATE_RC:-"true"}
-CONDA_DIR=${CONDA_DIR:-"/usr/local/conda"}
+VERSION="${VERSION:-"latest"}"
```

Comment:
ℹ️ Can we bump up patch versions for all the Features in respective `devcontainer-feature.json` files? 
The Features will be automatically published once this PR is merged. (Provided the version has changed)

Reply:
Done!